pulp_ansible/app/viewsets.py

```diff
@@ -409,6 +710,44 @@ class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet):
 
     parent_viewset = AnsibleRepositoryViewSet
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repository_model_or_obj_perms:ansible.view_ansiblerepository",
```

Comment:
It might make sense to inherit the permissions from the repo versions repository for this. if you have permissions to modify a repository, that should include it's versions.